docs/development/add_live_setting.rst

```diff
@@ -0,0 +1,15 @@
+Adding A New Live Setting
+==========================
+
+* Add new live setting to ``IndexSettings`` in ``luceneserver.proto`` and regenerate the protobuf files
```

Comment:
It would be good to mention that all new fields must be messages, with primitives being wrapped